EAD-Session-1-Assignment/functions.js

```diff
@@ -0,0 +1,373 @@
+/**
+ * Class containing methods to validate the form
+ */
+class Validation
+{
+    Validation(){}
+
+    /**
+     * Method to validate the user details
+     * @param {*} name is the user name
+     * @param {*} email is the user email
+     * @param {*} password is the user password
+     * @param {*} confirmPassword is the confirm password field
+     * @param {*} contact is the user contact number
+     * @param {*} gender is the user gender
+     */
+    static validateUserInformation(name , email , password , confirmPassword , contact , gender)
+    {
+        // Validating Contact Number
+        var count=0;
+		var filterContact = new RegExp('([a-zA-Z!@#$%^&*()_+])');
+		if(!(contact.length <= 10 && contact.length >= 8) ) {
+			this.errorDescription('required_e_contact',"Insert Number Between 8 to 10");
+			count++;
+		}else if(filterContact.test(contact)) {
+			this.errorDescription('required_e_contact',"Numeric values are only allowed");
+		} else {
+			this.removeDescription('required_e_contact',"");
+			count--;
+		}
+
+        //Validating user name
+		var filterName = new RegExp('\\d');
+		if(name.length <= 2) {
+			this.errorDescription('required_e_name',"Insert name greater then 2 word");
```

Comment:
Change message like username length can not be less than two characters.

Reply:
Done